### DISCUSSION TITLE
Open filename under mousecursor (via key + click)

### DISCUSSION THREAD

**Discussion by jerik** [kovidgoyal/kitty#3291] [Q&A]
Hi, 

iTerm has a usefull feature, which I try to get working in kitty. I could not find any information how to do this (perhaps I used the wrong keywords for search). 

When my mousecursor is over a file  I can open it in the default programm via key + click - see [example iTerm](https://dbader.org/blog/click-and-jump-to-any-file-or-folder-from-the-terminal). 

How can I achive this or a similar feature in kitty?

Best regards -- jerik

**Comment by kovidgoyal** [comment]
kitty has far more powerful capabilities

https://sw.kovidgoyal.net/kitty/kittens/hints.html
and 
https://sw.kovidgoyal.net/kitty/open_actions.html
